src/vibrator-sysfs.cpp

```diff
@@ -34,7 +34,12 @@ VibratorSysfs::VibratorSysfs(): Vibrator() {
 
     m_device = udev.device_from_syspath("/sys/class/leds/vibrator");
 
-    m_device.set_sysattr("trigger", "transient");
+    try {
+        m_device.set_sysattr("trigger", "transient");
+    } catch (const std::runtime_error& e) {
+        (void)e;
```

Comment:
Please omit setting the e variable instead of calling void on it.

Something like

```suggestion
    } catch (const std::runtime_error& /* e */) {
```